code/krasikova_ekaterina_lab1/view/dheap_view.py

```diff
@@ -0,0 +1,143 @@
+import tkinter as tk
+
+from krasikova_ekaterina_lab1.viewmodel import dheap_viewmodel
+
+
+class DHeapView:
+
+    view_model = dheap_viewmodel.DHeapViewModel()
+
+    def __init__(self):
+        self.root = tk.Tk()
+        self.root.title('D-Heap')
+        self.root.geometry('900x200')
+        self.root.resizable(width=False, height=False)
+
+        self.frame = tk.Frame(self.root)
+
+        self.d_label = tk.Label(self.root, text="D:")
+        self.d = tk.Entry(self.root, width=10)
```

Comment:
Please rename it to self.entry_param_d for example. This is necessary to better understand the meaning of the variable.